Assets/Managers/PrefabRepository.cs

```diff
@@ -0,0 +1,87 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+
+public class PrefabRepository : MonoBehaviour
+{
+    public float LoadingProgressPercentage { get; internal set; }
```

Comment:
Ye, this isn't in percentage it's goes from 0 to 1. What's that called? Fraction?

Reply:
Fraction works